index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="container">  
+  <header>
+    <div class="top-header"><img src="images/Time.png" alt="time">
+    <img src="images/wifi.png" alt="wifi">
+  </div>
+    <div class="bottom-header">
+      <h1>Welcome</h1>
+      <span class="identical-to">&#8801;</span>
+  </div>
+  </header> 
+  <main>
+    <div class="intro">
```

Comment:
- [ ] This whole section of code from the headings to the paragraph and to the Icons at the bottom must be wrapped in a `section` tag and not a `div` tag as this is the correct Html and CSS semantic for best practices.

refer to this [link for more information](https://www.w3schools.com/html/html5_semantic_elements.asp)

Reply:
You are right to the point, done!